Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLMeshObject.cs

```diff
@@ -26,6 +29,7 @@ public class RGLObject
     {
         public string Identifier;
         public RGLMesh RglMesh;
+        public int TextureID = 0;
```

Comment:
Why 0 is the default ID? I'd prefer to see the RGLTexture object here. We can check if it is assigned (not equal to null) and read its identifier.